Upstream: no
Reason: use local source instead of cmake fetching from git

--- a/cmake/par2-turbo.cmake
+++ b/cmake/par2-turbo.cmake
@@ -54,5 +54,1 @@ ExternalProject_add(
-	GIT_REPOSITORY	https://github.com/nzbgetcom/par2cmdline-turbo.git
-	GIT_TAG			v1.3.0
-	TLS_VERIFY		TRUE
-	GIT_SHALLOW		TRUE
-	GIT_PROGRESS	TRUE
+	URL					${PAR2_TURBO_URL}
